allauth/socialaccount/providers/eventbrite/provider.py

```diff
@@ -53,7 +53,6 @@ def extract_email_addresses(self, data):
                 EmailAddress(
                     email=email.get("email"),
                     verified=email.get("verfified"),
-                    primary=email.get("primary"),
```

Comment:
Eventbrite API does not a 'primary' attribute. This line causes `integrity error` during sign in